### PR TITLE
States fail to apply to openEuler 22.03 client

### LINKED ISSUE #7635: States fail to apply to openEuler 22.03 client
### Problem description

The initial states that are applied at registration time fail. This can also be seen when applying a highstate on a openEueler 22.03 client. The problem seems to be a wrong certificate path.

```
[root@vm-panaditas tmp]# venv-salt-call state.apply highstate
local:
    Data failed to compile:
----------
    No matching sls found for 'highstate' in env 'base'
[root@vm-panaditas tmp]# venv-salt-call state.apply 
[CRITICAL] Rendering SLS 'base:certs' failed: could not find expected ':'; line 8

---
[...]
    - name: rhn-org-trusted-ssl-cert


redhat.sls

mgr_proxy_ca_cert_symlink:    <======================
  file.symlink:
    - name: /usr/share/rhn/RHN-ORG-TRUSTED-SSL-CERT
    - target: /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT
    - onlyif: grep -Eq "^proxy.rhn_parent *= *[a-zA-Z0-9]+" /etc/rhn/rhn.conf && -e /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT

---
local:
    Data failed to compile:
----------
    Rendering SLS 'base:certs' failed: could not find expected ':'; line 8

---
[...]
    - name: rhn-org-trusted-ssl-cert


redhat.sls

mgr_proxy_ca_cert_symlink:    <======================
  file.symlink:
    - name: /usr/share/rhn/RHN-ORG-TRUSTED-SSL-CERT
    - target: /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT
    - onlyif: grep -Eq "^proxy.rhn_parent *= *[a-zA-Z0-9]+" /etc/rhn/rhn.conf && -e /etc/pki/trust/anchors/RHN-ORG-TRUSTED-SSL-CERT

---
```
There seems to be an easy solution, which I will propose as final in a PR, as easy as copying `/usr/share/susemanager/salt/certs/redhat.sls` to `/usr/share/susemanager/salt/certs/openeuler.sls`. As for now, the file contained the text `redhat.sls`. Making the file be a link to `redhat.sls` did not work either.

### Steps to reproduce

1. Register an openEuler client, with the right openEuler tools (as of today, from https://download.opensuse.org/repositories/systemsmanagement:/saltstack:/bundle:/next:/openEuler2203/openEuler_22.03/
2. Get the error during the initial states of the registration: "[Apply states [certs, channels, packages, services.salt-minion]"
3. Fix the error by correcting the file `/usr/share/susemanager/salt/certs/openeuler.sls`.
...

### Uyuni version

```bash
Information for package Uyuni-Server-release:
---------------------------------------------
Repository     : uyuni-server-stable
Name           : Uyuni-Server-release
Version        : 2023.09-230900.207.8.uyuni3
Arch           : x86_64
Vendor         : obs://build.opensuse.org/systemsmanagement:Uyuni
Support Level  : Level 3
Installed Size : 1.4 KiB
Installed      : Yes (automatically)
Status         : up-to-date
Source package : Uyuni-Server-release-2023.09-230900.207.8.uyuni3.src
Summary        : Uyuni Server
Description    : 
    Uyuni lets you efficiently manage physical, virtual,
    and cloud-based Linux systems. It provides automated and cost-effective
    configuration and software management, asset management, and system
    provisioning.
```

### Uyuni proxy version (if used)

_No response_

### Useful logs

```bash
Output already attached in the description.
```

### Additional information

I will send a PR with a proposed fix.

### PR DESCRIPTION
**Pull request by raulillo82**
## What does this PR change?

This PR fixes a wrong certificate for openEuler clients. Making the file be a symlink to redhat.sls was not working, and the original content of the name of the file did not work either. I think this was working in the first tests of the past though.
I'll be glad to hear about any other proposals.

## GUI diff

Before: Registration failed with the step called "Apply states [certs, channels, packages, services.salt-minion]"
After: Registration won't fail any more.

- [X] **DONE**

## Documentation
- No documentation needed: only internal and user invisible changes

- [X] **DONE**

## Test coverage
- No tests: already covered

- [X] **DONE**

## Links

Fixes #7635

- [X] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [ ] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

### CODE REVIEW

File: susemanager-utils/susemanager-sls/salt/certs/openeuler.sls
Comment (juliogonzalez) [inline susemanager-utils/susemanager-sls/salt/certs/openeuler.sls:1]: I don't think we need such `if`s? When is `osrelease` going to be 6 for openeuler?

File: susemanager-utils/susemanager-sls/salt/certs/openeuler.sls
Comment (juliogonzalez) [inline susemanager-utils/susemanager-sls/salt/certs/openeuler.sls:14]: Same

File: susemanager-utils/susemanager-sls/susemanager-sls.changes.raul.fix_7635
Comment (juliogonzalez) [inline susemanager-utils/susemanager-sls/susemanager-sls.changes.raul.fix_7635:1]:
```suggestion
- Fix the certificate management for openEuler
```
We need to specify what we are fixing.

File: susemanager-utils/susemanager-sls/salt/certs/openeuler.sls
Comment (raulillo82) [reply susemanager-utils/susemanager-sls/salt/certs/openeuler.sls:1]: This section indeed makes no sense. Removed.

File: susemanager-utils/susemanager-sls/salt/certs/openeuler.sls
Comment (raulillo82) [reply susemanager-utils/susemanager-sls/salt/certs/openeuler.sls:14]: Section removed.

File: susemanager-utils/susemanager-sls/susemanager-sls.changes.raul.fix_7635
Comment (raulillo82) [reply susemanager-utils/susemanager-sls/susemanager-sls.changes.raul.fix_7635:1]: Changelog changed.

**Review by juliogonzalez — APPROVED** [review]
LGTM, but wait for Victor's approval, in case I am missing something else :-D

**Comment by juliogonzalez** [comment]
Ping @vzhestkov

**Pull request closed without merging**
